### PR TITLE
Updated the Quicksilver Doc to Include a  Note emphasizing the Correct …

### LINKED ISSUE #6512: Automate and Integrate your WebOps Workflow with Quicksilver Doc Update
Re: [Automate and Integrate your WebOps Workflow with Quicksilver](https://pantheon.io/docs/quicksilver)

Priority: Low

## Issue Description:

I think it could be clearer on this page that, if the "web_docroot: true" setting is also set, how Quicksilver is affected.

The paths in the "pantheon.yml" should not have "web/" at the start of them, but Quicksilver will be looking for files inside the "web" folder. It is somewhat alluded to on this page, but customers often miss it.

## Suggested Resolution

If I have some time soon, I'll try to create a PR that does this, or I'm certainly happy to look over someone else's PR to make this a bit more clear for customers.

### PR DESCRIPTION
**Pull request by whitneymeredith**
…pantheon.yml script files path

Closes #6512 

## Summary

**[Automate and Integrate your WebOps Workflow with Quicksilver](https://pantheon.io/docs/quicksilver)** - I reworded the information on the file script paths for Quicksilver and created a note to emphasize the correct script paths for  pantheon.yml files.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7017](https://pr-7017--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `60` | `2` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7017` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/33003/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/33003/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7017--pantheon-docs.my.pantheonfrontend.website/docs)

**Review by joa-pan — APPROVED** [review]
Sorry, jumped the gun on this. Just had a tiny suggestion

**Review by dficker — COMMENTED** [review]
Here's some thoughts. I think it's an improvement in clarity, for sure, and I think a few more things could be improved, maybe.

File: source/content/quicksilver.md [hunk 1]
```diff
@@ -47,9 +47,15 @@ Quicksilver scripts that trigger on the deploy hook operate on the state of the
...
 
-We recommend setting up a dedicated directory in the docroot (e.g., `private/scripts`) for tracking these files. If your site uses a [nested docroot](/nested-docroot), the scripts directory needs to be located in the `web` subdirectory of your site's code repository (e.g., `web/private/scripts`), but the script paths in your `pantheon.yml` file should not include the `web/` path prefix and should resemble the examples above.
+Setting up a dedicated directory in the docroot (for example, `private/scripts`) for tracking these files is recommended. Setting the `web_docroot` to `true` instructs Quicksilver to look for files inside the `web` folder. If your site uses a [nested docroot](/nested-docroot), the scripts directory must be located in the `web` subdirectory of your site's code repository (for example, `web/private/scripts`). 
+
+<Alert type="info" title="Note">
+
+The script paths in your `pantheon.yml` file should not include the `web/` path prefix. Scripts in your `pantheon.yml` file should match the following path examples: `private/scripts/new_relic_deploy.php` or `private/scripts/slack_deploy_notification.php`
+
+</Alert>
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/quicksilver.md:58]: @dficker Thanks! I will take a closer look at this on Monday. Have a great weekend!

**Comment by whitneymeredith** [comment]
@dficker Let me know if these changes read better to you.

File: source/content/quicksilver.md [hunk 2]
```diff
@@ -47,9 +47,15 @@ Quicksilver scripts that trigger on the deploy hook operate on the state of the
 
 ## Script Type and Location
 
-Quicksilver currently supports `webphp` scripting, which runs a PHP script via the same runtime environment as the website itself. PHP scripts are subject to the same limits as any code on the platform, like [timeouts](/timeouts), and cannot be batched. In the future we may add additional types. The commands will run in order, and only execute the next when the previous has finished or timed out.
+Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website itself. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and will run in order, with each command executing after the previous command has finished or timed out. In the future we may add additional types.
```
Comment (joa-pan) [inline source/content/quicksilver.md:50]: I'm not sure, but I don't think we should mention things that we don't currently offer (and are not currently working on) such as "In the future we may add additional types. "

how about?...

```suggestion
Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and run continuously and sequentially. Each command executes after the previous command has finished or timed out.
```

**Comment by whitneymeredith** [comment]
@joa-pan Thanks - it reads great!

**Review by joa-pan — APPROVED** [review]
LGTM

**Pull request merged**